conftest.py

```diff
@@ -3,7 +3,19 @@
 from driver_wrapper import DriverWrapper
 from driver_selection import WebdriverSelection
 from config import URL, TIMEOUT, WEBDRIVER
-from utilities.db import prepare_db
+from utilities.db import prepare_db, restart_tomcat
+
+import allure
+from allure_commons.types import AttachmentType
+from time import sleep
+
+
+@pytest.fixture(scope='session', autouse=True)
+def prep_db(worker_id):
+    if worker_id == 'gw0' or worker_id == 'master':
```

Comment:
why not for all workers?

Reply:
if I run in parallel mode 'session' run for each thread